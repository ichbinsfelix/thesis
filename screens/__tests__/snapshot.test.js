import React from 'react'
import { render } from '@testing-library/react-native'
import Person from '../steps/2_profile'
import renderer from 'react-test-renderer'
import Purpose from '../steps/3.4_purpose'
import EnergyCharacteristics from '../steps/3.3_property_Energy_Characteristics'
import PropertyPrice from '../steps/3.2_property_Price'
import PropertyAdress from '../steps/3.1_property_adress'
it('renders', () => {
  render(<Person />)
})
it(`renders correctly Person view`, () => {
  const tree = renderer.create(<Person />)
  expect(tree).toMatchSnapshot()
})

it(`renders correctly Address view`, () => {
  const tree = renderer.create(<PropertyAdress />)
  expect(tree).toMatchSnapshot()
})

it(`renders correctly Price view`, () => {
  const tree = renderer.create(<PropertyPrice />)
  expect(tree).toMatchSnapshot()
})

it(`renders correctly Energy Characteristics view`, () => {
  const tree = renderer.create(<EnergyCharacteristics />)
  expect(tree).toMatchSnapshot()
})

it(`renders correctly Purpose view`, () => {
  const tree = renderer.create(<Purpose />)
  expect(tree).toMatchSnapshot()
})
