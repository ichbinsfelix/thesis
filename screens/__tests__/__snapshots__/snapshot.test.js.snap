// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Address view 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  testID="property-address"
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;

exports[`renders correctly Energy Characteristics view 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  testID="property-energy"
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;

exports[`renders correctly Person view 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  testID="profile-screen"
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;

exports[`renders correctly Price view 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  testID="property-price"
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;

exports[`renders correctly Purpose view 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  testID="purpose"
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</RCTSafeAreaView>
`;
